Dockerfile

```diff
@@ -1,57 +1,50 @@
-FROM node:18 AS builder
-
-RUN apt-get update && apt-get install -y \
-    php \
-    php-cli \
-    php-zip \
-    php-xml \
-    php-mbstring \
-    php-curl \
-    php-sqlite3 \
-    php-ldap \
-    unzip \
-    cron
-
-RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
-
+####################
+# Build image
+####################
+FROM node:23-alpine AS builder
 WORKDIR /app
 
-COPY . .
+ADD js/ ./js/
```

Comment:
`ADD` is recommended for remote files. For copying local files, it's recommended to use the `COPY` command.

https://docs.docker.com/build/building/best-practices/#add-or-copy